### DISCUSSION TITLE
Permission on Nested Documents

### DISCUSSION THREAD

**Discussion by manuelbiermann** [outline/outline#2902] [Ideas]
It would be awesome to be able to set Permissions on nested documents or to be able to create Sub-Collections with separate Permissions.

Without this, the number of collections needed to satisfy all privacy requirements can get out of hand fast.

Context:
I am considering moving the documentation for different project teams into outline but need to keep the access separate. This would lead in our case to ~50 additional collections if we can't bundle them in a "Projects" collection.

**Comment by dkkb** [comment]
In fact, I think this discussion is connected to https://github.com/outline/outline/discussions/2904.

There're many amazing features on `document` and `collection` including permission management, if we can combine them, we don't need to develop this feature again.

What are your thoughts?

🥺

**Reply by manuelbiermann** [reply]
Yeah, this sounds like a good way to structure it.
